pom.xml

```diff
@@ -6,7 +6,7 @@
     <version>2.2.0-SNAPSHOT</version>
   </parent>
   <artifactId>alien4cloud-kubernetes-plugin</artifactId>
-  <version>2.2.0-SNAPSHOT</version>
+  <version>2.2.0-SM10</version>
```

Comment:
I don't think this is a good idea to modify the version here. Useful for local test but maybe not on the remote

Reply:
Fixed